src/makeCalculator.js

```diff
@@ -36,6 +36,41 @@
  */
 function makeCalculator() {
   // write code here
+  const calc = {
+    result: 0,
+    add(a) {
+      calc.result += a;
+
+      return calc.result;
+    },
+    subtract(a) {
+      calc.result -= a;
+
+      return calc.result;
+    },
+    multiply(a) {
+      calc.result *= a;
+
+      return calc.result;
+    },
+    divide(a) {
+      calc.result /= a;
+
+      return calc.result;
+    },
+    reset() {
+      calc.result = 0;
+
+      return calc;
+    },
+    operate(callback, number) {
+      calc.result = callback(number);
```

Comment:
there is no need to assign the result of callback to calc.result again as you do this in the callback